utils/class_distribution.py

```diff
@@ -0,0 +1,90 @@
+import os
+import cv2
+import numpy as np
+from rich.console import Console
+from rich.table import Table
+from collections import defaultdict
+from typing import Dict
+import typer
+
+# We need to add the unetpp package to the system path so that we can import it
+import sys
+from pathlib import PurePath
+
+unetpp_path = PurePath(__file__).parent.parent
+sys.path.append(str(unetpp_path))
+
+from utils.preparation_data import CityScapesClasses
+
+TARGET_MASK_SUFFIX = "_mask.png"
```

Comment:
I have two suggestions:
1. Can you provide the suffix as a CLI parameter?
2. Instead of using name suffix `_mask` + file extension `.png`, can we just handle the suffix of the name?
    - check https://github.com/Korred/unet-pp/blob/main/unetpp/utils/images.py#L11

Reply:
you can check